style.css

```diff
@@ -0,0 +1,67 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+section{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: 100vh;
+    background: #042104;
+
+    animation: animateBg 10s linear infinite;
+}
+@keyframes animateBg{
+    0%{
+        filter: hue-rotate(0deg);
+    }
+    100%{
+        filter: hue-rotate(360deg);
+    }
+}
+section .loader{
+    position: relative;
+    width: 120px;
+    height: 120px;
+}
+section .loader span{
```

Comment:
Do we need section selection here actually??? Since loader is of higher specified i don't think so `section` will make a remarkable impact here!!! It will only make a larger selector which will eventually be difficult to read!

Reply:
yes, it's a larger one...
only for my simplicity, I added it

Reply:
yeah, done with these changes